src/components/autoFocuser.js

```diff
@@ -65,27 +65,32 @@ define(["focusManager", "layoutManager"], function (focusManager, layoutManager)
         candidates = candidates.concat(arrayFrom(container.querySelectorAll(".btnResume")));
         candidates = candidates.concat(arrayFrom(container.querySelectorAll(".btnPlay")));
 
-        var notFound = candidates.every(function (element) {
+        var focusedElement;
+
+        candidates.every(function (element) {
             if (focusManager.isCurrentlyFocusable(element)) {
```

Comment:
shouldn't you check here if `focusedElement` is still false-y?

Reply:
`return false` in callback stops `every`.